app/views/transport_requisitions/show.html.erb

```diff
@@ -0,0 +1,252 @@
+<div class="row wrapper border-bottom white-bg page-heading">
+  <div class="col-sm-8">
+    <h2>Transport Requisition: <strong><%= @transport_requisition.reference_number %></strong> </h2>
+    <ol class="breadcrumb">
+      <li>
+        <%= link_to 'Home', '/' %>
+      </li>
+      <li>
+        <%= link_to 'Transport Requisitions', '/en/transport_requisitions/?operation_id=' + @transport_requisition.operation_id.to_s %>
+      </li>
+      <li class="active">
+        <strong>Transport Requisition Detail</strong>
+      </li>
+    </ol>
+  </div>
+  <div class="col-sm-4">
+    <div class="title-action">
+      <%= link_to 'Edit', '/en/transport_requisitions/' + @transport_requisition.id.to_s + '/edit', class: 'btn btn-primary'  %>
+    </div>
+  </div>
+</div>
+
+<div class="wrapper wrapper-content row">
+  <div class="ibox col-sm-12">
+    <div class="ibox-content col-sm-12">
+      <div class="col-sm-1">
+        <% if (@transport_requisition&.status == 'open') %>
+          <span class="label label-warning"><%= @transport_requisition&.status %></span>
+        <% elsif (@transport_requisition&.status == 'closed') %>
+          <span class="label label-success"><%= @transport_requisition&.status %></span>
+        <% else %>
+          <span class="label label-warning"><%= @transport_requisition&.status %></span>
+        <% end %>
+      </div>
+      <div class="col-sm-5">
+        <div class="form-group col-md-12 col-sm-12 col-xs-12">
+          <div class="col-md-6 col-sm-6 col-xs-6 text-right">
+            Operation:
+          </div>
+          <div class="col-md-6 col-sm-6 col-xs-6">
+            <%= Operation.find(@transport_requisition.operation_id).name %>
```

Comment:
@nathnael same comment as above - logic in the view. I know there are incidents like these everywhere but let's stop doing this sort of code and we shall come back to refactor previously created ones.